README.md

```diff
@@ -108,6 +111,17 @@ Currently the scripts consider some events to be “changes”, “deploys”, a
 *   four\_keys.deployments
 *   four\_keys.incidents
 
+To update the scripts, we recommend that you update the sql files in the `queries` folder, rather than in the BigQuery UI.  Once you've edited the SQL, run the `schedule.py` script to update the scheduled query that populates the table.  For example, if you wanted to update the `four_keys.changes` table, you'd run:
+
+```sh 
+python3 schedule.py --query_file=changes.sql --table=changes
```

Comment:
Not sure if I should explicitly call `python3` in all the python calls in the README.  It really depends on how the user has their environment configured - if `python` points to `python3`, this is unnecessary, and you can just use `python`